<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseWindow</class>
 <widget class="QMainWindow" name="BrowseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnimexUI - Browse</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/medias/qrcs/logo.qrc">
    <normaloff>:/newPrefix/imgs/logo.png</normaloff>:/newPrefix/imgs/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 25, 25);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BrowseWindow_ExitBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: none;
	font-size: 19px;
	color:  rgb(255, 66, 69);
}

QPushButton::hover {
	background-color: rgb(255, 66, 69);
	color: white;
}</string>
    </property>
    <property name="text">
     <string>âœ–</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseWindow_LogoutBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: black;
	color: white;
	border: none;
	font-size: 19px;
	padding-top: 5px;
}

QPushButton::hover {
	background-color: grey;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="BrowseWindow_MainHeadingLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border-image: none;
	background: none;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Browse A Series.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BrowseWindow_MainTextLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border-image: none;
	background: none;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>You can browse a series either by clicking on its picture; or you can search it up by name.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="BrowseWindow_SearchLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(184, 184, 184);
	border: none;
	color: black;
	padding-top: 4px;
	padding-left: 20px;
	padding-right: 20px;
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Ex: Jojo</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="BrowseWindow_ScrollArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>801</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
	border: 1px solid rgb(230, 230, 230);
}

QScrollBar:vertical {
	border: none;
	background-color: black;
	border-radius: 5px;
	width: 15px;
	margin: 15px 0 0;
}


/* Handle */
QScrollBar::handle:vertical {
	background-color: rgb(255, 66, 69);
}
QScrollBar::handle:vertical::hover {
	background-color: rgb(255, 10, 22)
}

/* Top Btn */
QScrollBar::sub-line:vertical {
	background-color: grey;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>784</width>
       <height>500</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="BrowseWindow_SeriesHolderFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color: white;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/medias/qrcs/logo.qrc"/>
 </resources>
 <connections/>
</ui>
